content/master/operations/watchoperation.md

```diff
@@ -0,0 +1,600 @@
+description: WatchOperations create Operations when watched resources change
+---
+
+A `WatchOperation` creates [Operations]({{<ref "operation">}}) when watched
+Kubernetes resources change. Use WatchOperations for reactive operational
+workflows such as backing up databases before deletion, validating
+configurations after updates, or triggering alerts when resources fail.
+
+<!-- vale Google.Headings = NO -->
+## How WatchOperations work
+<!-- vale Google.Headings = YES -->
+
+WatchOperations watch specific Kubernetes resources and create new Operations
+whenever those resources change. The changed resource is automatically injected
+into the Operation for the function to process.
+
+```yaml
+apiVersion: ops.crossplane.io/v1alpha1
+kind: WatchOperation
+metadata:
+  name: config-validator
+spec:
+  watch:
+    apiVersion: v1
+    kind: ConfigMap
+    matchLabels:
+      validate: "true"
+  concurrencyPolicy: Allow
+  operationTemplate:
+    spec:
+      mode: Pipeline
+      pipeline:
+      - step: validate
+        functionRef:
+          name: function-config-validator
+        input:
+          apiVersion: fn.crossplane.io/v1beta1
+          kind: ConfigValidatorInput
+          rules:
+          - required: ["database.url", "database.port"]
+          - format: "email"
+            field: "notification.email"
+      - step: notify
+        functionRef:
+          name: function-slack-notifier
+        input:
+          apiVersion: fn.crossplane.io/v1beta1
+          kind: SlackNotifierInput
+          channel: "#alerts"
+          severity: "warning"
+```
+
+{{<hint "important">}}
+WatchOperations are an alpha feature. You must enable Operations by adding
+`--enable-operations` to Crossplane's arguments.
+{{</hint>}}
+
+## Key features
+
+- **Watches any Kubernetes resource type** - Not limited to Crossplane resources
+- **Supports namespace and label filtering** - Target specific resources
+- **Automatically injects changed resources** - Functions receive the triggering resource
+- **Configurable concurrency policies** - Control operation creation
+
+## Resource watching
+
+WatchOperations can watch any Kubernetes resource with flexible filtering:
+
+### Watch all resources of a type
+
+```yaml
+spec:
+  watch:
+    apiVersion: apps/v1
+    kind: Deployment
+```
+
+### Watch resources in a specific namespace
+
+```yaml
+spec:
+  watch:
+    apiVersion: v1
+    kind: ConfigMap
+    namespace: production
+```
+
+### Watch resources with specific labels
+
+```yaml
+spec:
+  watch:
+    apiVersion: example.org/v1
+    kind: Database
+    matchLabels:
+      backup: "enabled"
+      environment: "production"
+```
+
+### Watch cluster-scoped resources
+
+```yaml
+spec:
+  watch:
+    apiVersion: v1
+    kind: Node
+    matchLabels:
+      node-role.kubernetes.io/worker: ""
+```
+
+## Resource injection
+
+When a WatchOperation creates an Operation, it automatically injects the 
+changed
+<!-- vale write-good.TooWordy = NO -->
```

Comment:
i think this is causing a premature line break on https://deploy-preview-956--crossplane.netlify.app/master/operations/watchoperation/#resource-injection